resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "Seems to be ordering {PRONOUN/m_c/poss} fellow Clanmates around",
+            "Tries to put on a brave face for {PRONOUN/m_c/poss} Clanmates",
+            "Is feeling proud of {PRONOUN/m_c/self}",
+            "Is daydreaming about a Clan celebration in {PRONOUN/m_c/poss} honor",
+            "Is asking the Clan leader what {PRONOUN/m_c/subject} can do to help out around camp",
+            "Has been imitating the Clan leader's behaviour recently",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
+            "Offers to help {PRONOUN/m_c/poss} Clanmates with their duties",
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better"
+        ],
+        "main_trait_constraint": [
+            "ambitious"
+        ]
+    },
+    {
+        "id": "arrogant_gen",
+        "thoughts": [
+            "Is sure of {PRONOUN/m_c/poss} skills being better then anyone else",
+            "Is confidently spreading misinformation on something {PRONOUN/m_c/subject} {VERB/m_c/know/knows} little about",
+            "Is confidently spreading misinformation on something {PRONOUN/m_c/subject} {VERB/m_c/know/knows} nothing about",
+            "Is lecturing about something someone said wrong offhandedly",
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
+            "Took the lead during a patrol even though nobody wanted {PRONOUN/m_c/object} to",
+            "Has simply decided not to do what {PRONOUN/m_c/subject} were told",
+            "Is bragging to everyone"
+        ],
+        "main_trait_constraint": [
+            "arrogant"
+        ]
+    },
+    {
+        "id": "bloodthirsty_gen",
+        "thoughts": [
+            "Is acting suspicious",
+            "Has been washing {PRONOUN/m_c/poss} paws a lot lately",
+            "Growls to {PRONOUN/m_c/self}",
+            "Has disappeared often recently",
+            "Is practicing forbidden battle moves",
+            "Tells the warriors to get ready for a battle",
+            "Yearns to fight",
+            "Is letting a wound go untreated in hopes it gives {PRONOUN/m_c/object} a scar",
+            "Tried to fight a loner",
+            "Tried to fight a kittypet",
+            "Tried to fight a rogue",
+            "Wonders if training in the Dark Forest is really that bad",
+            "Daydreams about killing a certain cat",
+            "Wonders if {PRONOUN/m_c/subject} could kill a fox all by {PRONOUN/m_c/self}",
+            "Started a fight with a fellow Clanmate",
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better",
+            "Licks fresh-kill blood off {PRONOUN/m_c/poss} lips a little too intently",
+            "Has been glaring at a certain cat in camp with an odd look in {PRONOUN/m_c/poss} eye",
+            "Wonders how easy it'd be to attack the leader from where they lay",
```

Comment:
needs to be a separate thought group with random leader cat specified and pronoun tagged